src/cumulativeLine.ts

```diff
@@ -52,5 +58,5 @@ export function renderCumulativeLine(pareto: Pareto) {
         .attr("d", line)
         .style("fill", "none")
         .style("stroke", "#3050EF")
-        .style("stroke-width", "2");
+        .style("stroke-width", 3);

```

Comment:
Just a small thing,  I think we should avoid magic numbers and store this in a variable instead. Perhaps you can fix that, but I approve it now.

Reply:
I have change it now

Reply:
One minor comment, I would put the variable at the top together with the other variables, or even in the resources file

Reply:
It's on the top now